transfers.go

```diff
@@ -314,10 +314,11 @@ func createUserTransfers(custRepo customerRepository, depRepo depositoryReposito
 			return
 		}
 
-		id, userId, requestId := nextID(), moovhttp.GetUserId(r), moovhttp.GetRequestId(r)
+		userId, requestId := moovhttp.GetUserId(r), moovhttp.GetRequestId(r)
 		ach := achclient.New(userId, logger)
 
 		for i := range requests {
+			id := nextID()
```

Comment:
I think this fixes a bug where multiple transfers would have been created with the same ID.